cartography/intel/gcp/compute.py

```diff
@@ -333,6 +333,39 @@ def attach_compute_disks_to_instance_tx(
         gcp_update_tag=gcp_update_tag,
     )
 
+@timeit
+def attach_compute_networks_to_instance(session: neo4j.Session, data_list: List[Dict], instance_id: str, update_tag: int) -> None:
+    session.write_transaction(attach_compute_networks_to_instance_tx, data_list, instance_id, update_tag)
+
+
+@timeit
+def attach_compute_networks_to_instance_tx(
```

Comment:
Check `_attach_gcp_nics` method. we already done this in that

Reply:
done